frost-ed448/src/tests/deserialize.rs

```diff
@@ -30,12 +30,12 @@ fn check_deserialize_non_prime_order() {
             .try_into()
             .unwrap();
     let r = <Ed448Shake256 as Ciphersuite>::Group::deserialize(&encoded_point);
-    assert_eq!(r, Err(GroupError::InvalidNonPrimeOrderElement));
+    assert_eq!(r, Err(GroupError::MalformedElement)); // TODO: it should be `GroupError::NonPrimeOrderElement`
```

Comment:
I don't know why this test doesn't work, but everything else is fine. The `ed448_goldilocks_plus` code looks much better than the original library.

Reply:
`ed448_goldilocks_plus` has a slightly different implementation of `CompressedEdwardsY::decompress()`:
https://docs.rs/ed448-goldilocks/latest/src/ed448_goldilocks/curve/edwards/extended.rs.html#41-66
https://docs.rs/ed448-goldilocks-plus/latest/src/ed448_goldilocks_plus/curve/edwards/extended.rs.html#231-255

Reply:
Fixed in https://github.com/mikelodder7/Ed448-Goldilocks/pull/2